my.css

```diff
@@ -0,0 +1,163 @@
+.header {
+	margin: 0;
+	padding: 0;
+
+	background: black;
+	position: relative;
+}
+
+.logo {
+	position: absolute;
+	width: 98px;
+	height: 37px;
+	left: 52px;
+	top: 36px;
+	color: white;
+	font-family: Montserrat;
+	font-style: normal;
+	font-weight: 500;
+	font-size: 30px;
+	line-height: 37px;
+}
+
+.ru {
+	display: flex;
+	justify-content: space-around;
+	position: absolute;
+	width: 31px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)